### DISCUSSION TITLE
How to configure that the display power off after 10 minutes and power on on touch?

### DISCUSSION THREAD

**Discussion by cniweb** [raspiblitz/raspiblitz#3924] [Q&A]
I would like to configure that the display is going to sleep when nobody using the system.
One Wakeup event is when anybody uses the touchscreen.
Is there a preferred way?

**Comment by openoms** [comment]
There is no way to switch the screen off as it is lacking the physical switch to do that. 
Could program it to display only a black screen at best.
For programming look at the Blitz TUI.
Here is some documentation to get started: https://github.com/rootzoll/raspiblitz/blob/v1.7/home.admin/BlitzTUI/docs/tutorial.md

**Comment by NilByte** [comment]
Something like this might work.

https://github.com/timothyhollabaugh/pi-touchscreen-timeout

**Comment by rootzoll** [comment]
>Something like this might work.
>https://github.com/timothyhollabaugh/pi-touchscreen-timeout

@fusion44 had you any luck so far controling the LCD backlight? Maybe this script can help?

**Comment by openoms** [comment]
> Something like this might work.
> 
> https://github.com/timothyhollabaugh/pi-touchscreen-timeout

This is for the "Official Raspberry Pi touchscreen" : https://www.raspberrypi.org/products/raspberry-pi-touch-display/
 not the one we are using.

There is no way to control the backlight of this 3.5" screen without modifying the PCB.

**Comment by cniweb** [comment]
Is this possible (over GPIO 18):
https://www.waveshare.com/wiki/3.5inch_RPi_LCD_(C)#Use_GPIO_to_control_backlight_brightness
or this:
https://learn.adafruit.com/adafruit-pitft-3-dot-5-touch-screen-for-raspberry-pi/backlight-control

**Comment by fusion44** [comment]
@rootzoll 
While searching for solution I came to the conclusion that it was not possible due to hardware constraints. 
As openoms mentioned there are people talking about where we have to physically modify the hardware. 

@cniweb I think you need to solder something to the PCB of the screen for this method. I don't think that most of our users will be able to do that.

**Comment by ChuckNorrison** [comment]
you dont need to solder the pcb, just add a switch into the cable. not tested yet, but i would like to modify this behaviour too

**Comment by openoms** [comment]
I remember seeing a video where someone soldered a simple switch on the back of the PCB to toggle the power supply of the screen, but can't find it now.
